### ISSUE TITLE
Consider to use CSS sprites on users page

### ISSUE THREAD

**Issue by chuwy** [snowplow/snowplow.github.com#350]
Multiple images make [page](http://snowplowanalytics.com/product/users/) loading ~15sec. We can reduce http requests significantly by using CSS sprites. There's plenty of plugins for it nowadays.

**Comment by bogaert** [comment]
Makes sense. At some point I'd rather remove the whole page - it needs regular updating, and case studies and dedicated landing pages will do a better job creating leads. Thoughts @yalisassoon?

**Comment by alexanderdean** [comment]
I agree. I think a page which started as collating "social proof" for using Snowplow has drifted into us maintaining an open-access user database...
